INSTALL.md

```diff
@@ -1,115 +1,226 @@
+
+```bash
+  pushd ~/
   curl -O https://raw.githubusercontent.com/biocore/qiita/master/qiita_core/support_files/config_test.cfg
-  ```
+  popd
+```
 
-2. Set your `QIITA_CONFIG_FP` environment variable to point to that file:
+Set your `QIITA_CONFIG_FP` environment variable to point to that file (into `.bashrc` if using bash; `.zshrc` if using zshell):
 
-  ```bash
+```bash
   echo "export QIITA_CONFIG_FP=$HOME/config_test.cfg" >> ~/.bashrc
-  echo "export MOI_CONFIG_FP=$QIITA_CONFIG_FP" >> ~/.bashrc
+  echo "export MOI_CONFIG_FP=$HOME/config_test.cfg" >> ~/.bashrc
   source ~/.bashrc
-  ```
+  # Re-enable conda environment for qiita
+  source activate qiita
+```
+
+Setup ipython profile for qiita:
+
+```bash
+ipython profile create qiita-general --parallel
+```
+
+Next, make a test environment:
+
+```bash
+qiita-env make --no-load-ontologies
+```
 
-3. Start a test environment:
 
-  ```bash
-  qiita-env make --no-load-ontologies
-  ```
 
-4. Start the redis server:
-  ```bash
-  redis-server
-  ```
+## Starting Qiita
 
-5. Start the IPython cluster:
+Start postgres (Instructions vary depending on operating system and install method).
 
-  ```bash
-  qiita-env start_cluster qiita-general && sleep 30
-  ```
+Next, start redis server (the command may differ depending on your operating system and install location):
 
-6. Build the documentation (you may need to add `sudo` depending on your
-   privileges and the installation location:
+```bash
+redis-server
+```
 
-  ```bash
-  qiita pet webserver
-  ```
+Start the IPython cluster:
 
-7. Start the server:
+```bash
+qiita-env start_cluster qiita-general && sleep 30
+```
 
-    ```bash
-    qiita pet webserver start
-    ```
+Start the qiita server:
 
+```bash
+# this builds documentation before starting the server
+# alternatively: qiita pet webserver --no-build-docs start
+qiita pet webserver start
+```
 If all the above commands executed correctly, you should be able to go to http://localhost:21174 in your browser, to login use `test@foo.bar` and `password` as the credentials. (In the future, we will have a *single user mode* that will allow you to use a local Qiita server without logging in. You can track progress on this on issue [#920](https://github.com/biocore/qiita/issues/920).)
 
-## Installation issues on Ubuntu 14.04
+
+
+# Frequently Asked Questions and Troubleshooting
+
+As a first line of defense, ensure that you activated the qiita conda environment (assuming you're using conda as per the above instruction). If it wasn't previously active, try re-running your commands.
+
+```bash
+source activate qiita
+```
+
+**Q: Conda – What if I already have python installed? Or homebrew on OS  X? And a bunch of pip packages??**
+
+This isn't a problem because conda *prepends* its binaries to the `$PATH` variable.
+
+**Q: I get the error** `ConfigParser.NoOptionError: No option 'cookie_secret' in section: 'main'`
```

Comment:
If this is still happening is a bug in our code, and we should fix it rather than letting users deal with it.